{\rtf1\ansi\ansicpg1252\cocoartf949\cocoasubrtf430
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww25100\viewh13860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\fs24 \cf0 diff --git a/vendor/rails/activesupport/lib/active_support/test_case.rb b/vendor/rails/activesupport/lib/active_support/test_case.rb\
index 97b2b6e..3eafa6b 100644\
--- a/vendor/rails/activesupport/lib/active_support/test_case.rb\
+++ b/vendor/rails/activesupport/lib/active_support/test_case.rb\
@@ -35,5 +35,7 @@ module ActiveSupport\
     include ActiveSupport::Testing::Assertions\
     include ActiveSupport::Testing::Deprecation\
     extend ActiveSupport::Testing::Declarative\
+    include ActionController::TestProcess\
   end\
 end\
}